core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -221,13 +277,18 @@ public Set<String> getNamespaceTableNames(String namespace) throws ExecutionExce
   @Override
   public boolean namespaceExists(String namespace) throws ExecutionException {
```

Comment:
Ah, it looks like it's backward incompatible change. `namespaceExists()` won't work after upgrading Scalar DB until running the upgrade command. 

Ideally, `namespaceExists()` should work after upgrading Scalar DB even before running the upgrade command. But I'm wondering if we can do that. What do you think?

Reply:
Since we decided not to handle backward compatibility, I think the `namespaceExists()` logic is ok as is. Please have a look gain.